### PR TITLE
Add tests for compiling XRPackage from different file types

### PR DESCRIPTION
**Pull request by shu8**
This PR adds 3 tests for compiling XRPackages from a file (html, glb, vrm). Ensures:

- manifest `start_url` is correct
- asset content is correctly saved
- MIME type is correct

Relevant docs for these methods added in https://github.com/webaverse/docs/pull/22.

Part of #64.

### CODE REVIEW

File: tests/compile-from-file.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+};
+
+const pageFunction = async path => {
+  const blob = await fetch(path).then(res => res.blob());
+  blob.name = path.split('/').pop();
+
+  const xrpk = await XRPackage.compileFromFile(blob);
+
+  // Puppeteer only supports passing serializable data to/from Node
+  return JSON.stringify(xrpk);
```
Comment (avaer) [inline tests/compile-from-file.js:57]: How does this work? Will this be binary data (`Uint8Array`)?

If so, maybe we should consider a cleaner binary pipe that doesn't involve converting to a list of numbers.

Comment (shu8) [reply tests/compile-from-file.js:57]: Yeah it's a `Uint8Array`. I'm not sure what type of binary pipe we could use here?

Comment (avaer) [reply tests/compile-from-file.js:57]: base64?

Comment (shu8) [reply tests/compile-from-file.js]: I've pushed an update that uses `FileReader` to convert the `Uint8Array` to a base64 string, although I'm not sure if this is the best way?

Comment (avaer) [reply tests/compile-from-file.js]: There's many ways to do it but `FileReader` works. Thanks!

**Pull request merged**